src/Intro.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import key from './api/api.js';
+
+function Intro () {
+
+  let city = '';
+
+  function inputChange (e) {
+    city = e.target.value;
+  }
+
+  function getWeather () {
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${key}`)
+    .then(response => response.json())
+    .then(data => console.log('Success : ', data))
+    .catch(error => console.log('Error : ', error))
+  }
+  
+  function handleEnter (e) {
+    if (e.keyCode === 0) {
+      getWeather(city)
+    }
+  }
+  return (
+    <div className="background">
+      <div className="card">
```

Comment:
When using Material-UI, you never need to use divs again - you can use `Box`.

https://material-ui.com/components/box/

Similarly, CSS can all be handled easily by the `Box` component. Then you can get rid of your css file completely.

```suggestion
    <Box minHeight="100vh" bgColor="#d3f6f9" display="flex" justifyContent="center">
      <Box textAlign="center" position="absolute" top="20%" width="50%" padding="80px">
```

notice, since your CSS is in JS with Material-UI, you can even add constants, so your color could be assigned to a human-readable variable.



<Box

Reply:
I used the Box component with bgColor to assign the background color but the background did not change. I might import HUE from material-ui to deal with the color while still using the Box component